plugin/passwords.go

```diff
@@ -16,7 +16,7 @@ var (
 )
 
 type passwordGenerator interface {
-	GeneratePasswordFromPolicy(ctx context.Context, policyName string) (password string, err error)
+	GeneratePasswordFromPolicy(ctx context.Context, PasswordPolicy string) (password string, err error)
```

Comment:
nit: This name should probably be kept as it has the context of password generation and is consistent with other versions of this interface

Reply:
Done @pcman312